foundation/foundation-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -124,6 +124,14 @@
                         </dependency>
                     </dependencies>
                 </plugin>
+                <plugin>
+                    <groupId>org.technologybrewery.habushu</groupId>
+                    <artifactId>habushu-maven-plugin</artifactId>
+                    <configuration>
+                        <sourceFailOnLintErrors>true</sourceFailOnLintErrors>
```

Comment:
S: Aren't we setting this to true in the build parent already? We should avoid duplicate definitions as much as possible.

Reply:
Within the dev branch, the disable error on lint is set to False. The feature branch has removed these configurations from and will not have the sourceFailOnLintErrors/testFailOnLintErrors configured in the build-parent.